src/maintainer/updating_pkgs.rst

```diff
@@ -32,7 +32,11 @@ When a new version of a package is released on PyPI/CRAN/.., we have a bot that
 #. Manually setting up git remotes:
 
    - Clone the conda-forge feedstock repository
-   - Add the remote of the bot: ``git add regro-cf-autotick-bot git://github.com/regro-cf-autotick-bot/<package>-feedstock.git``
+   - Add the remote of the bot: ``git add regro-cf-autotick-bot git@github.com:regro-cf-autotick-bot/<package>-feedstock.git``
+
+     .. important::
+        It is not possible to push to a GitHub repository using the ``git://`` or ``https://``
```

Comment:
You can push using `https://`. Only `git://` is disallowed because there's no authentication.

Reply:
Fair enough. I've added more language because I'm [not](https://github.com/conda-forge/staged-recipes/issues/9522) [alone](https://github.com/conda-forge/conda-forge.github.io/issues/894#issuecomment-541342901) in being baffled when `https://` didn't work.